src/_includes/partials/meta-info.njk

```diff
@@ -1,8 +1,11 @@
 {% set pageTitle = title + ' - ' + site.name %}
 {% if socialImage %}
   {% set socialImage = site.url + socialImage %}
-{% else %}
```

Comment:
Oops. We no longer have a fallback for pages that have no `socialImage` and `noSocialImage` is set to true.

We should add the `else` statement.